core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -542,8 +542,8 @@ void preparedTxAlreadyExists(RecoveryAwareTransaction previous,
    void unableToLockToInvalidate(Object key, Address address);
 
    @LogMessage(level = ERROR)
-   @Message(value = "Execution error", id = 136)
-   void executionError(@Cause Throwable t);
+   @Message(value = "Error executing command %s, writing keys %s", id = 136)
+   void executionError(String commandType, Collection<Object> affectedKeys, @Cause Throwable t);
```

Comment:
It's not that hard to figure out what keys were affected from other logs, if TRACE is enabled... The problem is here that users don't enable TRACE and still want to know what keys might have become inconsistent because of a replication timeout.